README.md

```diff
@@ -8,104 +8,60 @@ git clone git@github.com:zeek/zeek-pkg-web.git
 cd zeek-pkg-web
 ```
 
-## Copy files/directories
-```
-sudo cp -a bropkg /var/www
-### Modify salt, database password, and CILogon client id/key in 
-###     secrets/.env to something suitable for your installation
-cp secrets/.env /var/www/bropkg/config/
-chmod 640 /var/www/bropkg/config/.env
-cd /var/www/bropkg
-composer update
-sudo chgrp -R apache /var/www/bropkg
-```
+## Edit secrets/.env
 
-## Set up HTTPD
-Edit /etc/httpd/conf.d/ssl.conf :
-```
-<VirtualHost _default_:443>
-DocumentRoot "/var/www/bropkg"
+`secrets/.env` has a set of variables for passwords and such that PHP will need
+to connect to the database and update the packages list from GitHub.
 
-...
+## (For production only) Initialize an SSL certificate
 
-<Directory />
-  Options FollowSymLinks
-  AllowOverride All
-</Directory>
+- Edit `cert_setup/ssl-update.sh` and set the `DOMAINS` and `EMAIL` values to
+  be sane for your installation.
+- Run the `cert_setup/init-certs.sh` script. This will generate a Let's Encrypt
+  certificate, store it in the location that nginx container will use, and add
+  a cron task to automatically update it.
```

Comment:
This seems to be needed in general, without running this my dev setup would not come up.